### ISSUE TITLE
Blurry icons on HI-DPI screens

### ISSUE THREAD

**Issue by yoGhurrt1** [hadouken/hadouken#187]
Simple, all icons in webui looks blury on Hi-DPI screens 
![clipboard07](https://cloud.githubusercontent.com/assets/13383620/9341836/88f99d6a-45f8-11e5-81f2-aae0602f87a7.jpg)

I know this is minor, but maybe it would be changed some day.

**Comment by vktr** [comment]
Going forward we might need to include our own icons in the web interface. The current icons are the standard uTorrent WebUI icons.

**Comment by yoGhurrt1** [comment]
I wanted to check what would happend when i'll replace standard icon that is in .png with 256x256 .png file (how it would compare on HiDPI screen and on 1680x1050 with 100% scaling)- but after replacing webui and restarting service, nothing happend. I don't see any other directory where icons would be. Did i miss something?

**Comment by vktr** [comment]
You need to extract the `C:/ProgramData/Hadouken/webui.zip` file somewhere, then replace the icons, then zip it again and replace the existing file.

**Comment by yoGhurrt1** [comment]
I've done this, that's why i wrote about replacing icons in webui.zip

**Comment by vktr** [comment]
Did you also restart the service?

**Comment by yoGhurrt1** [comment]
Yes, like i wrote in second post, i've done that too. I even checked in .zip if i really replaced that .png but everything looks fine. I've done also Ctrl+F5 and private mode to try. In final test, i moved webui.zip from directory to get 404 error and moved it back, but nothing changed. That's why i'm out of ideas where there could be another "cache" or sth that blocks from replacing icons.

**Comment by yoGhurrt1** [comment]
Ok, what i found after some another experiments
- it doesn't metter on what icon you'll change with original one - even i used different icon from original webui.zip, icon on web didn't change
- all configurations of webui are stored in .zip so after every modification and service restart, interface should change
- after many restarts, hadouken service had a problem with itself, and instead of Program Data, he looked for webui in "Program Files". Reinstalling hadouken resolved temporarly this problem.
